packages/core/protocols/src/proto/dxos/client/services.proto

```diff
@@ -223,15 +223,16 @@ enum NetworkMode {
   ONLINE = 1;
```

Comment:
Change to ConnectionState { OFFLINE, ONLINE }
and variable from `mode` to `state`